dbmanager/dbmanager.h

```diff
@@ -0,0 +1,66 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+
+/*  datatype of each node in the database.
+ *  @key - a unique integer value.
+ *  @val - pointer to the data.
+ *  @left, @right - pointers to children.
+ */
```

Comment:
Ideally you want to hide all internal structures, i.e. dbNode should be defined within the source component.